app/views/admin/posts/index.html.haml

```diff
@@ -0,0 +1,128 @@
+- content_for(:page, 'posts')
+- content_for(:title, I18n.t("admin.navigation.messages"))
+
+%section#content
+
+  .page.page-form
+    .row
+      .col-md-6
+        .card
+          .card-header
+            %i.fas.fa-fw.fa-stream
+            = I18n.t("admin.navigation.messages")
+
+        #posts
+          - @posts.each do |post|
+            .card
+              = link_to post, class: 'card-header ' + ('avatar-padding' if post.author&.avatar&.attached?).to_s do
+                - if post.author&.avatar&.attached?
+                  = image_tag post.author.avatar_representation, class: 'avatar'
+
+                .card-img-overlay
+                  - if post.published?
+                    %span.ribbon.success
+                      = I18n.t("admin.posts.status.published")
+                  - if post.draft?
+                    %span.ribbon
+                      = I18n.t("admin.posts.status.draft")
+                  - if post.review?
+                    %span.ribbon
+                      = I18n.t("admin.posts.status.review")
+                  - if post.scheduled?
+                    %span.ribbon.warning
+                      = I18n.t("admin.posts.status.scheduled")
+
+                  - if post.pinned?
+                    %span.ribbon
+                      = I18n.t("admin.posts.status.pinned")
+
+                %h4= post.author.name
+                %ul
+                  - if post.published_at.present?
+                    %li
+                      = I18n.l(post.published_at, format: "%d %B %H:%m")
+
+                  - post.tags&.each do |tag|
+                    %li= tag
+
+              .card-body.content
+                = sanitize post.content, tags: %w(strong em u s h1 h2 h3 h4)
+
+
+        = render partial: 'layouts/partials/pagination', locals: {pagy: @pagination}
+
+
+      .col-md-6
+        = form_for @post, :html => { :class => 'form-validation post-content' } do |f|
```

Comment:
Make this a partial

Reply:
The partial will only be used there then. I can't make a partial and use it on the member side and the admin side. They are different.